qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -112,6 +110,25 @@
             driver_test_name_balloon = "balloon"
             iozone_cmd_opitons = " -azR -r 64k -n 512M -g 1G -M -I -i 0 -i 1 -b iozone.xls -f %s:\testfile"
             read_rng_cmd = "WIN_UTILS:\\random_%PROCESSOR_ARCHITECTURE%.exe"
+            x86_64:
+                # virtio-mem is not supported on 32-bit systems
+                # and viomem is supported from 1.9.40-0
+                boot_with_viomem = yes
+                driver_test_name_viomem = "viomem"
+                maxmem_mem = 80G
+                mem_devs = 'vmem0'
+                vm_memdev_model_vmem0 = "virtio-mem"
+                size_mem_vmem0 = 8G
+                requested-size_memory_vmem0 = 1G
+                memdev_memory_vmem0 = "mem-vmem0"
+                requested-size_test_vmem0 = "4G 8G 0"
+                test_drivers += ' viomem'
+                driver_test_name_viomem = "viomem"
+            i386:
+                # mainly to cover win10.i386 test without viomem
+                boot_with_viomem = no
+            Host_RHEL.m7, Host_RHEL.m8, Host_RHEL.m9.u0, Host_RHEL.m9.u1, Host_RHEL.m9.u2, Host_RHEL.m9.u3, Host_RHEL.m9.u4:
+                boot_with_viomem = no
```

Comment:
It was defined in above.

Reply:
Actually in my understanding it is not conflict，the first ‘boot_with_viomem = no’ is for i386.
the second if for both x86_64 and i386.